python/bindings.cpp

```diff
@@ -310,8 +313,10 @@ PYBIND11_MODULE(libsonata, m)
         )
         .def(
             "flatten",
-            &Selection::flatten,
-            "List of IDs constituting Selection"
+            [](Selection& obj) {
```

Comment:
3s -> 0.34s for 10^8 ids.
Replacing `reserve(N)` + `push_back` with `std::vector X(N)` + `std::iota` in `flatten()` C++ implementation reduces it further to 0.26s, but I guess these fractions of second are irrelevant.

Reply:
nice